scripts/format_notebooks.py

```diff
@@ -181,6 +181,58 @@ def preprocess(self, nb, resources):
         return nb, resources
 
 
+class VersionCheckPreprocessor(preprocessors.Preprocessor):
+    metadata_tag = "VersionCheck"  # tag for added cells so that we can find them easily
+
+    try:
+        # determine the current stellargraph version by importing the library's version file
+        sys.path.append(
+            (
+                os.path.abspath(
+                    os.path.join(
+                        os.path.dirname(__file__), os.path.pardir, "stellargraph"
+                    )
+                )
+            )
+        )
+        import version as sgversion
+
+        stellargraph_version = sgversion.__version__
+    except ImportError as error:
+        print(f"ERROR: unable to import StellarGraph version: {error.message}")
```

Comment:
It's a bit unusual to me that an error would just be printed, and execution would continue; what's the reasoning?

Reply:
changed now to the above code which is a little different; agree that it's better to just have the exception if the file isn't found